boards/arm64/qemu_cortex_a53/qemu_arm64_kvm.yaml

```diff
@@ -0,0 +1,13 @@
+identifier: qemu_arm64_kvm
```

Comment:
I do not understand why this board lives under `qemu_cortex_a53`. KVM virt uses the host CPU type and there is no guarantee that the host CPU type will be Cortex-A53.

If any, you should create a dedicated directory for this board (e.g. `boards/arm64/qemu_kvm_arm64`).